tools/codesigningtool/codesigningtool.py

```diff
@@ -108,7 +110,7 @@ def _parse_mobileprovision_file(mobileprovision_file):
 def _certificate_fingerprint(identity):
   """Extracts a fingerprint given identity in a mobileprovision file."""
   _, fingerprint, _ = execute.execute_and_filter_output([
-      "openssl",
+      "/usr/bin/openssl",
```

Comment:
I was curious about using `/usr/bin/openssl` v.s. running whatever is on the env? We may need to think about backwards compat: e.g. if people had a different openssl binary and now this method picks up the system one.

Maybe longer term we'd want to pass these as an input.

Reply:
I agree. I would rather keep it un-anchored for now.

Reply:
I'll put it back. This came from a discussion with Tim to ensure the Apple system one and not some alternate shim or copy from brew. But if folks have any reason to override that I guess we should respect it, it makes sense to me.

cc @timsutton

Reply:
Done